fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -938,9 +963,9 @@ public void updateDatatable(final String datatableName, final JsonCommand comman
             final boolean isConstraintApproach = this.configurationDomainService.isConstraintApproachEnabledForDatatables();
 
             if (!StringUtils.isBlank(entitySubType)) {
-                final String updateLegalFormSQL = "update x_registered_table SET entity_subtype='" + entitySubType
+                String updateLegalFormSQL = "update x_registered_table SET entity_subtype='" + entitySubType
                         + "' WHERE registered_table_name = '" + datatableName + "'";
-                this.jdbcTemplate.execute(updateLegalFormSQL); // NOSONAR
+                this.jdbcTemplate.execute(updateLegalFormSQL);
```

Comment:
Please don't remove the NOSONAR note from here.

Reply:
Done!